server/game/cards/01_SOR/units/FirstLegionSnowtrooper.ts

```diff
@@ -0,0 +1,22 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { KeywordName } from '../../../core/Constants';
+
+export default class FirstLegionSnowtrooper extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '4619930426',
+            internalName: 'first-legion-snowtrooper',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addConstantAbility({
+            title: 'While attacking a damaged unit, this unit gets +2/+0 and gains Overwhelm. (Deal excess damage to the opponent\'s base.)',
```

Comment:
We typically don't copy the text verbatim, just enough to communicate to the player what's about to happen (often times this text will be displayed on a button or something). In this case you could just remove the overwhelm explainer text

Reply:
I fixed the text to adhere to the standard. Thank you for the explanation.